polaris-core/src/main/java/org/apache/polaris/core/persistence/bootstrap/SchemaOptions.java

```diff
@@ -31,6 +32,17 @@ public interface SchemaOptions {
   @Nullable
   String schemaFile();
 
+  @Value.Default
+  default boolean setupSchema() {
+    return true;
+  }
+
+  @NotNull
+  @Value.Default
+  default String schemaName() {
+    return "POLARIS_SCHEMA";
```

Comment:
Isn't this kind of asymmetric now? `SchemaOptions` allow callers to set any schema name, but persistence implementations do not use it for actually creating the schema :thinking:  ... or did I miss it?